containers/seqkit/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/biocontainers/seqkit:0.7.1--0
+
+ARG VERSION=0.7.1--0
+
+LABEL container.base.image="quay.io/biocontainers/seqkit:0.7.1--0"
+LABEL software.name="seqkit"
+LABEL software.version=${VERSION}
+LABEL software.description="seqkit is a toolkit for FASTA/Q file manipulation"
+LABEL tags="Genomics"
+
+
+ADD cleanup_normalized_fastq.sh /scripts/
+
+WORKDIR /scratch
```

Comment:
You'll need to create a new repository in AWS ECR so that the new image can be stored. This is controlled in the tbsequencing-infrastructure repository, ecr.tf file.